Resources/Locale/en-US/station-events/events/second-life.ftl

```diff
@@ -0,0 +1,16 @@
+second-life-announce = A group of staff has boarded the station
+second-life-announce-dead = Second Life Ghost roles have opened up: { $description }
+
+second-life-tourist-announce = What we thought was great salvage was instead full of tourists! They're eager to come aboard, get them doing useful work before they attempt to leave.
+second-life-security-announce = The security training session has finally ended. They now know how to turn on their batons and are keen to test their new skills.
+second-life-engineer-announce = We've redirected some rogue engineers to the station. Please keep them so busy with technical issues that they forget to drink, let alone leave.
+second-life-service-announce = Star-Chain Accomidations has gone bankrupt. We've managed to hire some of their service staff at bargain rates. Expect them soon.
+second-life-medical-announce = The recent star-plague has passed, all the patients have died. Fortunately that means we found some spare medical staff ready for easier work.
```

Comment:
```suggestion
second-life-service-announce = Star-Chain Accommodations has gone bankrupt. We've managed to hire some of their service staff at bargain rates. Expect them soon.
```

dont think this company is referenced anywhere else so it might read as kind of weird. maybe make it more generic? not sure

Reply:
oop, didnt mean to remove medical one, rider review is annoying sometimes

Reply:
Okay, I just made it up so yeah it's not referenced.